.bingo/Variables.mk

```diff
@@ -18,43 +18,43 @@ GO     ?= $(shell which go)
 #
 GOJSONTOYAML := $(GOBIN)/gojsontoyaml-v0.0.0-20200602132005-3697ded27e8c
 $(GOJSONTOYAML): .bingo/gojsontoyaml.mod
-	@# Install binary/ries using Go 1.14+ build command. This is using bwplotka/bingo-controlled, separate go module with pinned dependencies.
+	@# Install binary/ries using Go 1.16+ build command. This is using bwplotka/bingo-controlled, separate go module with pinned dependencies.
```

Comment:
This is a generated file. No need to touch it manually.

Reply:
got it, thanks